src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -145,6 +145,38 @@
     </ItemGroup>
   </Target>
 
+  <!--
+    Overriding VerifyClosure target to enable closure validation without other validations.
```

Comment:
If this is just temporary rather than forking the whole thing, just have a single target:
```xml
<Target Name="EnsureDepenedencyGraphDir" BeforeTargets="VerifyClosure">
    <MakeDir Directories="$(PackageReportDir)$(Id)" />
</Target>
```

Reply:
I actually need to override the target to update it's condition as well, to run if new property is set - `ShouldVerifyClosure`. Currently `VerifyClosure` only runs if `SkipValidatePackage` is false: https://github.com/dotnet/arcade/blob/e11b8486ad155fab13e788a268755c0ca60d3609/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/framework.packaging.targets#L170

Reply:
Is there a reason we can’t get the arcade update instead?  Would it take too long?  Could you manually update just the package with this fix?

Reply:
I've heard that there is a freeze on whole Arcade update, but I'll ask if a single package would be allowed.

Reply:
Heard back - we can get update in RC2. I will update `VerifyClosure` target in `Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk` in Arcade, and remove the target override from my PR. But, I will pick your suggestion for a new target to create `Reports` directory, which works around an issue in `VerifyClosure` task: https://github.com/dotnet/arcade/issues/6090. The task is in a different Arcade package and they want to minimize the number of changes now.

Reply:
@ericstj I will also move your suggested new target into Arcade next to VerifyClosure target. We can, then, remove this target when VerifyClosure task is fixed - with a single PR in Arcade.

Reply:
Will keep `EnsureDepenedencyGraphDir` in runtime repo - `PackageReportDir` is runtime-specific property, it doesn't make sense for this to be in Arcade.

Reply:
Arcade fix is in: https://github.com/dotnet/arcade/pull/6101

This (runtime) fix can be merged and closure validation would start working once we pick up new Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk package.

@ericstj @jkoritzinsky this is safe to merge once you approve the changes.